### DISCUSSION TITLE
[Enhancement] Add additional weather providers

### DISCUSSION THREAD

**Discussion by TylerWilliamson** [TylerWilliamson/QuickWeather#169] [Ideas]
I received an email requesting that I add [Weatherbit](https://www.weatherbit.io/) and [AerisWeather](https://www.aerisweather.com/develop/api/) as providers to QuickWeather. Please use this thread to suggest additional weather providers.

**Comment by shortwavesurfer2009** [comment]
You mean like NOAA?

**Comment by TylerWilliamson** [comment]
NOAA does not provide a data feed that I know of. I am refering to services with weather APIs like [OpenWeatherMap](https://openweathermap.org) and [Open-Meteo](https://open-meteo.com) and the others mentioned in the comment

**Comment by shortwavesurfer2009** [comment]
I thought NOAA was what wX was using but i could be wrong

https://gitlab.com/joshua.tee/wx

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by TylerWilliamson** [comment]
Oh, that app is indeed using the NOAA / NWS weather API directly. Both OpenWeatherMap and Open-Meteo use that model for US weather, but it does not seem to have weather forecasts for other countries. For now, I do not think I will be adding support for using the NOAA / NWS API directly

**Comment by shortwavesurfer2009** [comment]
Fair enough, i just thought the had one so wanted to mention it

**Comment by shed40** [comment]
Just to make people aware that openweathermap are making some changes to there API, they are migrating onecall 2.5 to onecall 3.0, and onecall 2.5 will close in june 2024.
You can sign up to onecall 3.0 and still get 1000 calls/day but you need to add your details for billing additional calls.

**Comment by TylerWilliamson** [comment]
@shed40 Thank you. I am already working on an update for QuickWeather which will inform users who use OneCall 2.5 that they will need OneCall 3.0 soon. QuickWeather will continue to work with OneCall 2.5 until OpenWeatherMap shuts down that API, but as of QuickWeather 2.6.10, they will be unable to enter new OpenWeatherMap API Keys that do not have OneCall 3.0

**Comment by ryester19** [comment]
Surprised Pirate Weather hasn't been mentioned at all yet. They even have alerts

https://pirateweather.net

**Comment by TylerWilliamson** [comment]
Pirate Weather seems very interesting. I will look into adding support for it
